robotools/tests.py

```diff
@@ -465,6 +465,153 @@ def test_parameter_validation(self) -> None:
         evotools._prepare_aspirate_dispense_parameters(
             rack_label="WaterTrough", position=1, volume=15, forced_rack_type="valid forced rack type"
         )
+
+        # define a labware correctly for testing purposes
+        plate = liquidhandling.Labware("DWP", 8, 12, min_volume=0, max_volume=2000, initial_volumes=100)
+        # test labware argument checks
+        with self.assertRaises(ValueError):
```

Comment:
```suggestion
        with self.assertRaises(ValueError, matches="required parameter: labware"):
```

With these kwargs you can make sure that it's not passing because _another_ `ValueError` is raised instead of the one you want to test